tests/integrations/web.py

```diff
@@ -2,13 +2,15 @@
 from src.masonite.broadcasting import Broadcast
 from src.masonite.authentication import Auth
 
-Route.get("/", "WelcomeController@show").name("welcome"),
-Route.post("/", "WelcomeController@show"),
-Route.post("/upload", "WelcomeController@upload").name("upload"),
-Route.get("/test", "WelcomeController@test"),
-Route.get("/emit", "WelcomeController@emit"),
-Route.get("/view", "WelcomeController@view"),
-Route.get("/mail", "MailableController@view"),
+ROUTES = [
+    Route.get("/", "WelcomeController@show").name("welcome"),
+    Route.post("/", "WelcomeController@show"),
+    Route.post("/upload", "WelcomeController@upload").name("upload"),
+    Route.get("/test", "WelcomeController@test"),
+    Route.get("/emit", "WelcomeController@emit"),
+    Route.get("/view", "WelcomeController@view"),
+    Route.get("/mail", "MailableController@view"),
+]
 
 Broadcast.routes()
```

Comment:
With this new approach what about those:
```python
Broadcast.routes()
Auth.routes()
```
How to handle this ? Should we do somehting like : 
```python
ROUTES += Brodcast.routes()
ROUTES += Authroutes()
```

Reply:
@josephmancuso what about this ?